qiskit/synthesis/two_qubit/two_qubit_decompose.py

```diff
@@ -1068,18 +1076,19 @@ def decomp1(self, target):
         return U1r, U1l, U0r, U0l
 
     def decomp2_supercontrolled(self, target):
-        """Decompose target ~Ud(x, y, z) with 2 uses of the basis gate.
+        r"""Decompose target :math:`\sim U_d(x, y, z)` with 2 uses of the basis gate.
+
+        For supercontrolled basis :math:`\sim U_d(pi/4, b, 0)`, all b, result :math:`U_r` has trace
 
-        For supercontrolled basis ~Ud(pi/4, b, 0), all b, result Ur has trace
         .. math::
 
-            |Tr(Ur.Utarget^dag)| = 4cos(z)
+            \Big\vert Tr(U_r \dot U_{target}^\dag)| = 4cos(z)
 
-        which is the optimal approximation for basis of CNOT-class ``~Ud(pi/4, 0, 0)``
-        or DCNOT-class ``~Ud(pi/4, pi/4, 0)`` and any target.
-        May be sub-optimal for b!=0 (e.g. there exists exact decomposition for any target using B
-        ``B~Ud(pi/4, pi/8, 0)``, but not this decomposition.)
-        This is an exact decomposition for supercontrolled basis and target ``~Ud(x, y, 0)``.
+        which is the optimal approximation for basis of CNOT-class :math:`\sim U_d(pi/4, 0, 0)`
```

Comment:
pi should be \pi

Reply:
I think some of these changes overlap with the ones I did in https://github.com/Qiskit/qiskit/pull/11719, so maybe it will be better to merge it first?

Reply:
yes, I added the on hold label to wait until #11719.

Reply:
ok, just merged the changes from your PR, we can remove the "on hold" and pi is now \pi.